### DISCUSSION TITLE
Permission Cache with Redis does not seems to work?

### DISCUSSION THREAD

**Discussion by khangphung-vl** [spatie/laravel-permission#2735] [Q&A]
Hi everyone,
I already successfully setup Redis cache for spatie permission, as I **enter the page as an User with role**, a new Redis key-value is added with the key is in the following format
`redis_prefix + cache_prefix + "spatie.permission.cache"`
Which seems to be correct (per the documentation description). But the value of the inserted key is `a:3:{s:5:"alias";a:0:{}s:11:"permissions";a:0:{}s:5:"roles";a:0:{}}` (which is a PHP serialized), and **can be translated** to 
`{
    "alias": [],
    "permissions": [],
    "roles": []
}`

Basically, it is empty. What is going on and is this an expected behavior?
